website_breadcrumb/README.rst

```diff
@@ -0,0 +1,93 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
```

Comment:
pls use new readme structure https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Reply:
A better link for this would be https://github.com/OCA/maintainer-tools/tree/master/template/module/readme. The new README will be autogenerated on merge (although you can pre-generate it if you want with OCA/maintainer-tools installed locally).